lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,2054 @@
+{
```

Comment:
<p>To převádění zase na json (kvůli zobrazení) mi přijde spíš zavádějící. Zobrazovat Python objekty typu list slovníků umíme i bez jsonu ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>